docs/data/query.md

```diff
@@ -202,6 +202,31 @@ See the [MongoDB Atlas Documentation](https://www.mongodb.com/docs/atlas/data-fe
 
 For information on connecting to your Atlas instance from other MQL clients, see the MongoDB Atlas [Connect to your Cluster Tutorial](https://www.mongodb.com/docs/atlas/tutorial/connect-to-your-cluster/).
 
+#### Query by date
+
+When using MQL to query your data by date or time range, you can optimize query performance by avoiding the MongoDB `$toDate` expression, using the [BSON `date` type](https://www.mongodb.com/docs/manual/reference/bson-types/#date) instead.
+
+For example, you could use the following to query a dataset over a date range, specifying a target start timestamp in BSON date format, and using the `Date()` constructor to use the current time as the end timestamp:
+
+```mongodb {class="command-line" data-prompt=">"}
+// Switch to sensorData database:
+use sensorData
+
+// Set desired start and end times:
+const startTime = 'Mon Feb 10 2024 14:45:07 GMT-0500 (Eastern Standard Time)'
```

Comment:
Ah I just noticed that this is a string type, not a date. That might confuse users. I don't really know JS but maybe we want something like:

```js
const startTime = new Date('Mon Feb 10 2024 14:45:07 GMT-0500 (Eastern Standard Time)')
```

Reply:
Ah, you're right I think, and we can simplify it a bit too if we're using `Date()` then. Thanks!